packages/constructs/test/evaluate-cfn.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * Simple function to evaluate CloudFormation intrinsics.
+ *
+ * Note that this function is not production quality, it exists to support tests.
+ */
+
+export function evaluateCFN(object: any, context: {[key: string]: string} = {}): any {
```

Comment:
Just noticing this moved as well.

We should keep all CloudFormation-related things out of `constructs`, no? This is our chance of re-introducing the proper conceptual split we kind of slacked on over the last year and a half :).

Reply:
This is just for the test. There shouldn’t be any cfn related stuff otherwise